sample/bootstrap-ads.yaml

```diff
@@ -28,3 +28,14 @@ static_resources:
         port_value: 18000
     http2_protocol_options: {}
     name: xds_cluster
+layered_runtime:
+  layers:
+    - name: runtime-0
+      rtds_layer:
+        rtds_config:
+          api_config_source:
```

Comment:
Shouldn't this be something like `ads: {}` instead? Can you also bump up to two layers in ADS only if possible?

Reply:
Oops!!! checking! My bad

Reply:
Addressed! Sorry it took so long, got pulled into something